Handler/handler_stock.go

```diff
@@ -60,21 +61,23 @@ func (h *StockHandler) UpdateStock(c *fiber.Ctx) error {
 	if err := c.BodyParser(stock); err != nil {
 		return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{"error": err.Error()})
 	}
-	err = h.UseCase.UpdateStock(stock.ToStock(), uint(id))
+	stockEntity, err := h.UseCase.UpdateStock(stock.ToStock(), uint(id))
 	if err != nil {
 		return c.Status(fiber.StatusInternalServerError).JSON(fiber.Map{"error": err.Error()})
 	}
-	return c.JSON(stock)
+	ResStock := payload.StockToStockRes(stockEntity)
+	return c.JSON(ResStock)
```

Comment:
last return in func handler it no status ?

 c.JSON(<data>)  >>>  c.Status(<status>).JSON(<data>)

Reply:
Fixed it. Please check again.